### PR TITLE
[Merged by Bors] - Postgres Source Connector

### PR DESCRIPTION
**Pull request by simlay**
Subtask of #106

### CODE REVIEW

File: rust-connectors/sources/postgres/src/connect.rs [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+        let admin = fluvio.admin().await;
+
+
+        let topics = admin.list::<TopicSpec, _>(vec![]).await?;
+        let topic_exists = topics.iter().any(|t| t.name == config.common.fluvio_topic);
+        if !topic_exists {
+            return Err(Error::TopicNotFound(config.common.fluvio_topic.to_string()).into());
+        }
+
+        let consumer = fluvio
```
Comment (morenol) [inline rust-connectors/sources/postgres/src/connect.rs:54]: could you move this to fluvio-connector-common? I mean just like we have create_producer() there

Comment (morenol) [reply rust-connectors/sources/postgres/src/connect.rs:54]: Or I can just do that in a separate PR

Comment (simlay) [reply rust-connectors/sources/postgres/src/connect.rs:54]: Oh, right. Good catch.

**Pull request closed without merging**